### PR TITLE
Add Pinpoint metrics and alarms

### PR DESCRIPTION
**Pull request by sastels**
# Summary | Résumé

Need to replicate the SNS / SMS alarms and dashboards

## Related Issues | Cartes liées

* https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/368

# Test instructions | Instructions pour tester la modification

Tested on dev (turning on `cloudwatch_enabled` so they'd get created)

# Release Instructions | Instructions pour le déploiement

None.

# Reviewer checklist | Liste de vérification du réviseur

- [ ] This PR does not break existing functionality.
- [ ] This PR does not violate GCNotify's privacy policies.
- [ ] This PR does not raise new security concerns. Refer to our GC Notify Risk Register document on our Google drive.
- [ ] This PR does not significantly alter performance.
- [ ] Additional required documentation resulting of these changes is covered (such as the README, setup instructions, a related ADR or the technical documentation).

> ⚠ If boxes cannot be checked off before merging the PR, they should be moved to the "Release Instructions" section with appropriate steps required to verify before release. For example, changes to celery code may require tests on staging to verify that performance has not been affected.

### CODE REVIEW

File: env/dev/pinpoint_to_sqs_sms_callbacks/terragrunt.hcl [hunk 1]
```diff
@@ -39,6 +40,7 @@ inputs = {
   sqs_deliver_receipts_queue_arn                     = dependency.common.outputs.sqs_deliver_receipts_queue_arn
   pinpoint_to_sqs_sms_callbacks_ecr_repository_url   = dependency.ecr.outputs.pinpoint_to_sqs_sms_callbacks_ecr_repository_url
   pinpoint_to_sqs_sms_callbacks_ecr_arn              = dependency.ecr.outputs.pinpoint_to_sqs_sms_callbacks_ecr_arn
+  pinpoint_monthly_spend_limit                       = dependency.common.outputs.sns_monthly_spend_limit
```
Comment (ben851) [inline env/dev/pinpoint_to_sqs_sms_callbacks/terragrunt.hcl:43]: So we're using the sns monthly limit as the pinpoint limit as well? Assuming these are two different figures, should we sum the two costs to get the overall SMS limit?

Comment (sastels) [reply env/dev/pinpoint_to_sqs_sms_callbacks/terragrunt.hcl:43]: 🤷  That's probably a good idea. Just keep one alarm but make it a combined one.

Comment (sastels) [reply env/dev/pinpoint_to_sqs_sms_callbacks/terragrunt.hcl]: ok for now I left the existing SNS alarms alone but made the new pinpoint alarm a combined alarm. Once we're happy that the new alarm is correct we can get rid of the old alarm

**Review by ben851 — APPROVED** [review]
LGTM!

**Pull request merged**